fed/config.py

```diff
@@ -48,8 +49,8 @@ def __init__(self, raw_bytes: bytes) -> None:
             self._data = cloudpickle.loads(raw_bytes)
 
     @property
-    def grpc_metadata(self):
-        return self._data.get(fed_constants.KEY_OF_GRPC_METADATA, {})
+    def cross_silo_comm_config(self):
+        return self._data.get(fed_constants.KEY_OF_CROSS_SILO_COMM_CONFIG, {})
```

Comment:
Default value is dict while config is a dataclass?

Reply:
you are right, fixed it.